k8s/Makefile

```diff
@@ -223,9 +224,11 @@ vendor:
 docker-build:
 	cd .. && \
 	if docker build --help | grep -q load; then \
-		docker build --load -f k8s/Dockerfile . -t $(IMG); \
+		docker build --load -f k8s/Dockerfile . \
+		-t $(IMG); \
```

Comment:
Keep this part untouched. No need to add a newline here.

Reply:
Done.